components/nexus-ehcache/src/main/java/org/sonatype/sisu/ehcache/CacheManagerComponentImpl.java

```diff
@@ -48,8 +50,8 @@
   private CacheManager cacheManager;
 
   @Inject
-  public CacheManagerComponentImpl() throws IOException {
-    this(null);
+  public CacheManagerComponentImpl(final ApplicationDirectories directories) throws IOException {
+    this(directories.getAppDirectory("etc/ehcache.xml"));
```

Comment:
ya, we could improve this helper to have get*File() variants but I want to wait a bit before adding stuff like that, as I think we need nio helpers too.

But ya, the component as-is was meant to return directories (and if we ever added sanity checks to ensure they are directories if they exist, things would blow up).